pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -144,7 +144,7 @@ func TestNodeStatusWithCloudProviderNodeIP(t *testing.T) {
 		Spec:       v1.NodeSpec{},
 	}
 
-	// TODO : is it possible to mock kubelet.validateNodeIP() to avoid relying on the host interface addresses ?
+	// TODO : is it possible to mock validateNodeIP() to avoid relying on the host interface addresses ?
```

Comment:
You could have an interface with `ValidateNodeIP` that would be implemented by your mocker and `Kubelet`